tests/helpers/linux/test_net.py

```diff
@@ -19,6 +20,11 @@
 
 
 class TestNet(LinuxHelperTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pid = os.getpid()
```

Comment:
I just pushed 77b9d3ad9819617698c5aa619ac5b73fba6921b3 with the change to `setUpClass` we discussed, so you can use `prog` here now if you'd like (although I don't think it's that bad to call `os.getpid()`/`find_task()` everywhere that you need it just for some test cases).

Reply:
Great!  I'll do it for `net` in `test_net.py`.